### ISSUE TITLE
Unable to prevent submenu collapse

### ISSUE THREAD

**Issue by jonathantyates** [Polymer/core-menu#5]
Currently, there is no way prevent the submenu collapsing. It would be nice if all submenus could all be opened at the same time.

**Comment by novabyte** [comment]
Yep, it'd be great if we could keep a submenu permanently open. I'd like to help is someone can give direction on the way they'd like it to be added?

**Comment by clyfe** [comment]
@novabyte I made this: https://gist.github.com/clyfe/ff9d9608ae2a35af18d2

**Comment by novabyte** [comment]
@clyfe Thanks! :smiley:
